crates/ethereum/reth/src/lib.rs

```diff
@@ -21,7 +21,7 @@ pub mod primitives {
 
 /// Re-exported cli types
 #[cfg(feature = "cli")]
-pub use reth_ethereum_cli as cli;
+pub use reth_ethereum_cli::cli::Cli as cli;
```

Comment:
we don't want this, pls revert. the issue means to export `reth_ethereum_cli::Cli` type from the bin crate and remove the type from there